src/renderer/containers/map/infoBox/infoBox.scss

```diff
@@ -34,6 +34,19 @@
   justify-content: end;
```

Comment:
Also change the css ".handicap-section" to have justify property to be "flex-end" instead of "end". 
Without that In chrome its rendering on the left  (works fine with justify-content as "end" in FF and Safari). and with the "Report Inactive" link they stack on top of each other.
![image](https://user-images.githubusercontent.com/1335655/64129609-1c45de80-cd8b-11e9-843b-9abd4c58ed71.png)